npc/cities/niflheim.txt

```diff
@@ -208,7 +207,11 @@ niflheim,350,258,1	script	Cursed Spirit#nif	4_NFWISP,{
 	next;
 	switch(select("Touch the first book.", "Touch the second book.", "Touch the third book.", "Okay, I am leaving.")) {
 	case 1:
-		monster "niflheim",349,259,"Rideword",1478,1,"Cursed Spirit#nif::OnMyMobDead";
+		if (.last_mob_summon - gettimetick(2) > 60) { // only spawn every 60 seconds, to prevent farming exploits
+			killmonster("niflheim", "Cursed Spirit#nif::OnMyMobDead");
+			.last_mob_summon = gettimetick(2);
+			monster("niflheim", 349, 259, "Rideword", 1478, 1, "Cursed Spirit#nif::OnMyMobDead");
```

Comment:
```suggestion
			killmonster("niflheim", strnpcinfo(3)+"::OnMyMobDead");
			.last_mob_summon = gettimetick(2);
			monster("niflheim", 349, 259, "--ja--", RIDEWORD, 1, strnpcinfo(3)+"::OnMyMobDead");
```

Reply:
`strnpcinfo(3)` should be `strnpcinfo(NPC_NAME_UNIQUE)` though
this isn't rathena